Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,39 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            var name = "Joe";
```

Comment:
Read input from conosle. Refer to `Console.ReadLine();`, int.Parse, float.Parse

Reply:
I did not read instructions carefully, should be fixed now